### ISSUE TITLE
29311099: Mail.app: accessibility label localization missing for swipe actions

### ISSUE THREAD

**Issue by openradar-mirror** [lionheart/openradar-mirror#16366]
#### Description

In the mail list view, when swiping to the left, the accessibility labels for the three action buttons read in German:

"NSSwipeActionItemMore Taste"
"NSSwipeActionItemMove Taste"
"NSSwipeActionItemTrash Taste"

Please fix.

-
Product Version: 10.1
Created: 2016-11-17T15:56:29.141370
Originated: 2016-11-17T16:56:00
Open Radar Link: http://www.openradar.me/29311099
